flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBResourceContainer.java

```diff
@@ -82,30 +89,36 @@ public final class RocksDBResourceContainer implements AutoCloseable {
 
     @VisibleForTesting
     public RocksDBResourceContainer() {
-        this(new Configuration(), PredefinedOptions.DEFAULT, null, null, false);
+        this(null, new Configuration(), PredefinedOptions.DEFAULT, null, null, false);
```

Comment:
I feel it really strange that we put the `null` in the first place, can we put the nullable parameter in the last places.